crates/cibse_guide_e/src/chapter_7/equation_7_9/openfire_runner.rs

```diff
@@ -0,0 +1,215 @@
+pub mod integration_tests;
+
+use framework::method::calculation::Calculation;
+use framework::method::calculation::CalculationComponent;
+use framework::method::form::{Form, FormStep};
+use framework::method::parameter::ArcParameter;
+use framework::method::parameter::ParameterTrait;
+use framework::method::parameter::ParameterValue;
+use framework::method::parameter::builder::ParamBuilder;
+use framework::method::parameters::Parameters;
+use framework::method::runner::MethodRunner;
+use framework::method::tag::Tag;
+use framework::method::test::IntegrationTests;
+use framework::method::validation::ParameterError;
+use framework::method::{Method, step::Step};
+
+use crate::CIBSEGuideE;
+
+use std::sync::{Arc, RwLock};
+use std::vec;
+
+struct Symbols {
+    n_in: &'static str,
+    t: &'static str,
+    w_e: &'static str,
+    p: &'static str,
+    a: &'static str,
+    s: &'static str,
+}
+
+const SYMBOLS: Symbols = Symbols {
+    n_in: "N_{in_{max}}",
+    w_e: "W_e",
+    t: "t",
+    p: "p",
+    a: "A",
+    s: "S",
+};
+
+#[derive(Default)]
+pub struct Chapter7Equation9Runner;
+
+impl MethodRunner for Chapter7Equation9Runner {
+    fn name(&self) -> String {
+        "Acceptance capacity of the stair".to_string()
```

Comment:
```suggestion
        "Acceptance capacity of a stair".to_string()
```

Small fish, do what you want here. I find `the` to be strangely assertive for a title. It feels like you are talking about a specific stair.

Reply:
Addressed.